test/unit/Reflector/GenericTest.php

```diff
@@ -160,6 +160,8 @@ public function testReflectWithAggregateSourceLocatorWhenIdentifierDoesNotExist(
 
     public function testReflectWithAggregateSourceLocatorFindsClass()
     {
+        $this->markTestIncomplete('See https://github.com/Roave/BetterReflection/issues/43');
```

Comment:
Please add further information to #43 as unsure as to the relevance? #43 is (currently) about naming as I see it